docs/UserGuide.md

```diff
@@ -173,37 +174,65 @@ Examples:
 * `view nm/John` returns the record for the student named `john`
 * `view nm/john alice` returns the records for the students named `John` and the student named `Alice`.
 
-### 3.7 Toggling view : `toggleView`
+### 4.2.2 Toggling view : `toggleView`
 
 *To be updated*
 
-### 3.8 Clearing all student records : `clear`
+### 4.2.3 Viewing all student records : `viewAll`
 
-Clears all student records from local storage.
+Shows a list of all students in the class.
+For each student in the list, only the Student's Name and Student's ID are displayed.
 
-Format: `clear`
+<div markdown="span" class="alert alert-info">:information_source:
+   **Note:** The other details of a student are hidden to reduce cluttering.
+To view the full record of a student, use the `view` command instead.
+</div><br>
 
-### 3.9 Help : `help`
+Format: `viewAll`
 
-Shows a summary of all commands available.
+### 4.2.4 Viewing student records from a class : `viewClass`
 
-Format: `help`
+Shows a list of all students in the specified class.
 
-### 3.10 Exiting the application : `exit`
+Format: `viewClass [class name]`
+
+<div markdown="span" class="alert alert-info">:information_source:
+   **Note:** Class name is case-insensitive.
+</div><br>
+
+Examples:
+* `viewClass 2A`
+* `viewClass Loyalty1`
+
+## 4.3 Exam statistics
+
+### 4.3.1 Getting exam statistics: `statistics`
```

Comment:
Is this intended? Or should it be `viewStats` as we discussed previously? Otherwise, we can fix this in a future PR.

Reply:
I think we'll confirm the list of command words again.